README.md

```diff
@@ -44,14 +44,14 @@ class MyClass {
 
 This logs:
 
-<pre>
+```sql
 
       create table student(
         id int primary key
         name text
       )
```

Comment:
```suggestion
create table student(
  id int primary key
  name text
)
```

Reply:
@jridgewell this code block is supposed to show that the output has an unwanted indentation - it shouldn't be "fixed" like this. I think we should revert this edit. Unless I'm missing something?

Reply:
Whoops. I'll submit a PR to fix.